### PR TITLE
added conversion_rate column in invoice table and created field for s…

### PR DESCRIPTION
**Pull request by bharatsingh100**
…ame in edit invoice view

Targets #814

### Description
Added the conversion rate field in the edit invoice page for international payments and added the column for the same in invoice table 

### Checklist:

- [ ] I have performed a self-review of my own code.

### CODE REVIEW

File: Modules/User/Resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -1,8 +1,8 @@
 @extends('layouts.app')
 @section('js_scripts')
-<script src="{{ mix('/js/user.js') }}" defer></script>
+<script src="{{ asset('/js/user.js') }}" defer></script>
```
Comment (GauravGusain98) [inline Modules/User/Resources/views/layouts/master.blade.php:3]: Is there any specific reason you have used `asset` here? We use `mix` for importing the js and CSS. Please try to use `mix` here.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Left some feedbacks. Please correct them. And whenever you are requesting for review remember to add `ready for review` label to the PR.

File: database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
+<?php
```
[the remark this answers is not in the document]
Comment (rathorevaibhav) [reply database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php:1]: @bharatsingh100 you can create a migration inside a module using:
```php
php artisan module:make-migration nameOfMigration moduleName
```

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php [hunk 3]
```diff
@@ -138,6 +138,13 @@
                     </div>
                 </div>
 
+                <div class="form-group" v-if="this.client.type != 'indian'">
```
[the remark this answers is not in the document]
Comment (bharatsingh100) [reply Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:141]: ok i will make the changes

File: database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (bharatsingh100) [reply database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php:1]: sure. thanks

File: database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('invoices', function (Blueprint $table) {
+            $table->text('conversion_rate')->nullable()->after('conversion_rate_diff');
```
[the remark this answers is not in the document]
Comment (bharatsingh100) [reply database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php:17]: ok will change this too

File: Modules/User/Resources/views/layouts/master.blade.php (hunk 1 above)
Comment (bharatsingh100) [reply Modules/User/Resources/views/layouts/master.blade.php:3]: i was not able to load user management page on my local setup, so anmol told me to change this to asset , then it worked.

Comment (bharatsingh100) [reply Modules/User/Resources/views/layouts/master.blade.php:3]: i will change it to mix again

File: database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (bharatsingh100) [reply database/migrations/2021_09_29_190515_add_conversion_rate_column_in_invoices_table.php:17]: conversion_rate_diff is of type "text" too, should i change it too

File: Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (bharatsingh100) [reply Modules/Invoice/Resources/views/subviews/edit/invoice-details.blade.php:141]: there are other places where it is hard coded too, should i change those too

**Review by GauravGusain98 — APPROVED** [review]
Looks good on code.

**Pull request merged**